packages/app/src/client/services/AdminCustomizeContainer.js

```diff
@@ -429,4 +444,71 @@ export default class AdminCustomizeContainer extends Container {
     }
   }
 
+
+  async deleteLogo() {
+    try {
+      const formData = {
+        _csrf:  this.appContainer.csrfToken,
+        attachmentId: this.state.attachmentId,
+      };
+      await this.appContainer.apiPost('/attachments.removeBrandLogo', formData);
+      this.setState({
+        isUploadedLogo: false,
+        uploadedLogoSrc: DEFAULT_LOGO,
+        attachmentId: null,
+        isDefaultLogo: true,
+      });
+
+    }
+    catch (err) {
+      this.setState({ retrieveError: err });
+      logger.error(err);
+      throw new Error('Failed to delete logo');
+    }
+  }
+
+  async uploadAttachment(file) {
```

Comment:
I think it would be better to change this method's name to something that tells us it's used only for uploading attachment for brand logo because the internal code looks like it is.

Reply:
Rename uploadAttachment to uploadBrandLogo